docs/metrics/ga.md

```diff
@@ -107,6 +107,24 @@ Here are the current events on the website as of this writing, grouped by their
 | Retire from Test Pilot | button click | Retire |
 | Shows upgrade notice | Upgrade Warning | upgrade notice shown |
 
+#### `Featured Experiment`
```

Comment:
Since this is always happening on the homepage, each of these should include `cd1`, `cd2`, and `cd3`. Everything should include `cd10`. Since each of these operate on a specific featured experiment, they should all include `cd11`.

Reply:
:+1: